shelley/chain-and-ledger/executable-spec/test/Rules/TestLedger.hs

```diff
@@ -93,3 +102,11 @@ consumedEqualsProduced = withTests (fromIntegral numberOfTests) . property $ do
 
           (balance u  + d  + fees  + foldl (+) (Coin 0) rewards ) ==
           (balance u' + d' + fees' + foldl (+) (Coin 0) rewards')
+
+
+-- | Transform LEDGER `sourceSignalTargets`s to DELEG ones.
+ledgerToDelegSsts
```

Comment:
"Ssts"

Reply:
@uroboros just to be sure I understand - are you suggesting 

```suggestion
ledgerToDelegSSTs
```

?